selenium-webdriver/ie.d.ts

```diff
@@ -7,12 +7,15 @@ declare namespace ie {
      */
     class Driver extends webdriver.WebDriver {
         /**
+         * Creates a new session for Microsoft's Internet Explorer.
+         *
          * @param {(capabilities.Capabilities|Options)=} opt_config The configuration
          *     options.
          * @param {promise.ControlFlow=} opt_flow The control flow to use,
          *     or {@code null} to use the currently active flow.
+         * @return {!Driver} A new driver instance.
          */
-        constructor(opt_config?: webdriver.Capabilities | Options, opt_flow?: webdriver.promise.ControlFlow);
+        static createSession(opt_config?: webdriver.CreateSessionCapabilities | Options, opt_flow?: webdriver.promise.ControlFlow): Driver;
```

Comment:
The implementations of some of the other createSession methods accept either capabilities or a `{desired, required}` hash. Should we match the JSDoc for those or the implementation?